components/prism-factor.js

```diff
@@ -84,12 +84,16 @@
 
 		// R/ regexp?\/\\/
 		'regexp': {
-			pattern: /(^|\s)R\/\s+(?:\\\S|[^\\/])*\//,
+			pattern: /(^|\s)R\/\s+(?:\\\S|[^\\/])*\/(?:[idmsr]*|[idmsr]+-[idmsr]+)(?=\s|$)/,
 			lookbehind: true,
 			alias: 'number',
 			inside: {
 				'variable': /\\\S/,
-				'keyword': /[+?*\[\]^$(){}.|]/
+				'keyword': /[+?*\[\]^$(){}.|]/,
+				'operator': {
+					pattern: /(\/)[idmsr]+(-[idmsr]+)?/,
```

Comment:
This will also match: `R/ \/i/`.
```suggestion
					pattern: /(\/)[idmsr]+(-[idmsr]+)?$/,
```

Reply:
There's already a passing test against this, and I'm pretty sure the reason it works as-is, is as you said above, `variable` and `keyword` will have consumed the `\/` and not cause `operator` to match `/i` (with lookbehind) in that case.

I can add the end-of-string assertion anyway though.

Reply:
Ah true. I didn't think of the `variable` pattern.

> I can add the end-of-string assertion anyway though.

Not necessary, the test is enough.